bin/dfx-network-deploy

```diff
@@ -49,7 +49,7 @@ if [[ "$DFX_NETWORK" == "local" ]]; then
 
   dfx start --clean --background
   dfx-nns-install --ic_commit "$DFX_IC_COMMIT"
-  dfx-nns-import
+  dfx-nns-import --network "$DFX_NETWORK"
```

Comment:
The effect of remote entries in dfx.json has changed in dfx v 0.14.1; e.g. __default remote IDs were previously a last resort and did not prevent deploying a canister.  Now it does, so adding a __default remote as before can cause deployments to break.  It seems best to be very specific about for which network canister IDs are wished.  Perhaps this is over-cautious and we can relax this but I would rather limit the scope until we have a clearer picture of how canister IDs will be managed in future.